packages/react/src/component-tests/IcEmptyState/IcEmptyState.cy.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import { mount } from "cypress/react";
+import React from "react";
+import {
+  BasicEmptyState,
+  EmptyStateTitle,
+  ButtonEmptyState,
+  EmptyStateAligned,
+  EmptyStateImage,
+} from "./IcEmptyStateTestData";
+import { IcEmptyState } from "../..";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
```

Comment:
Your visual tests are currently failing, which is probably because this component is so Text heavy (difference in font rendering between devices) - i'd recommend boosting this value to 0.05 and seeing what happens :) 

If needs be, you can add to the threshold on a per-test basis